vm-setup/roles/v1aX_integration_test/templates/clusterctl_env_template_centos.rc

```diff
@@ -32,23 +32,6 @@ export CTLPLANE_KUBEADM_EXTRA_CONFIG="
 {% if CAPM3_VERSION == "v1alpha3" %}
       - ifup eth1
 {% endif %}
-      - dnf update -y
-      - dnf install -y ebtables socat conntrack-tools
-      - dnf install python3 -y
-      - dnf config-manager --add-repo https://download.docker.com/linux/centos/docker-ce.repo
-      - setenforce 0
-      - sed -i 's/^SELINUX=enforcing$/SELINUX=permissive/' /etc/selinux/config
-      - dnf install docker-ce docker-ce-cli --disableexcludes=kubernetes --nobest -y
-      - >-
-        dnf install gcc kernel-headers kernel-devel keepalived
-        device-mapper-persistent-data lvm2 -y
-      - echo  \"Installing kubernetes binaries\"
-      - curl -L --remote-name-all https://storage.googleapis.com/kubernetes-release/release/v1.18.0/bin/linux/amd64/{kubeadm,kubelet,kubectl}
-      - chmod a+x kubeadm kubelet kubectl
-      - mv kubeadm kubelet kubectl /usr/local/bin/
-      - mkdir -p /etc/systemd/system/kubelet.service.d
-      - retrieve.configuration.files.sh \"https://raw.githubusercontent.com/kubernetes/release/{{ KUBERNETES_BINARIES_CONFIG_VERSION }}/cmd/kubepkg/templates/latest/deb/kubelet/lib/systemd/system/kubelet.service\" /etc/systemd/system/kubelet.service
-      - retrieve.configuration.files.sh \"https://raw.githubusercontent.com/kubernetes/release/{{ KUBERNETES_BINARIES_CONFIG_VERSION }}/cmd/kubepkg/templates/latest/deb/kubeadm/10-kubeadm.conf\" /etc/systemd/system/kubelet.service.d/10-kubeadm.conf
```

Comment:
you need to do something similar for the workers

Reply:
Ow, good catch, it might improve the timing more

Reply:
done